core/scheduler/service/replay_service.go

```diff
@@ -18,6 +18,31 @@ const (
 	getReplaysDayLimit = 30 // TODO: make it configurable via cli
 
 	metricJobReplay = "jobrun_replay_requests_total"
+
+	// list of supported plugins in optimus, which supports separate execution project.
+	// currently serves as a workaround to explicitly list execution project config keys in each plugins.
+	// after plugin standardization, this all will be removed
+	bq2bq           = "bq2bq"
+	bq2pg           = "bq2pg"
+	pg2bq           = "pg2bq"
+	bq2api          = "bq2api2"
+	clevertap       = "clevertap-bq-sync"
+	transporterTask = "transporter_task"
+)
+
+var (
+	pluginExecutionProjectKeyNameMap = map[string]string{
+		bq2bq:           "EXECUTION_PROJECT",
+		bq2pg:           "BQ_EXECUTION_PROJECT",
+		pg2bq:           "BQ_EXECUTION_PROJECT",
+		bq2api:          "EXECUTION_PROJECT",
+		clevertap:       "EXECUTION_PROJECT",
+		transporterTask: "EXECUTION_PROJECT",
+	}
+
+	namespaceConfigForReplayMap = map[string]map[string]string{
```

Comment:
do we need to make this as a map? since we only aim for replay execution project. unless we have another configs to be added.

Reply:
after separate discussion, will remove the map to increase readability in the config override function